activesupport/lib/active_support.rb

```diff
@@ -121,6 +121,12 @@ def self.to_time_preserves_timezone=(value)
       )
     end
 
+    if value && ![:zone, :offset].include?(value)
+      ActiveSupport.deprecator.warn(
```

Comment:
If this is a deprecation we need to tell which value will be used, and when the deprecated behavior will be removed.

Reply:
Yup, will get this updated.

Reply:
The deprecation warning is updated in https://github.com/rails/rails/commit/f538bbbea7f858ad324d288bc3399438f8cac06b